src/components/Filter/Category/Category.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import CategoryMenuItem from './CategoryMenuItem';
+import './category.css';
+
+const menuItems = [
+    {
+        label: 'Appliances',
+        count: '2448',
+        childrenItems: [
+            { label: 'Air Purifier', count: '54' },
+            { label: 'Fans', count: '59' },
+            { label: 'Handheld & Stick Vacuum', count: '123' },
+        ],
+    },
+    {
+        label: 'Audio',
+        count: '123',
+        childrenItems: [
+            { label: 'Audio1', count: '123' },
+            { label: 'Audio2', count: '123' },
+            { label: 'Audio3', count: '123' },
+        ],
+    },
+];
+
+const Category: React.FC = () => {
+    const { t } = useTranslation();
+    return (
```

Comment:
```suggestion
    const { t } = useTranslation();
    
    return (
```
Thêm line trống giữa các thành phần